deps/uv/src/unix/darwin.c

```diff
@@ -190,8 +190,12 @@ int uv_cpu_info(uv_cpu_info_t** cpu_infos, int* count) {
   }
 
   size = sizeof(cpuspeed);
+#if defined(__arm64__)
+  cpuspeed = 0;
```

Comment:
this is a hacky workaround for now. `hw.cpufrequency` is not present on ARM Macs as of right now. This was causing `os.cpus()` to return an empty array, which prevented the add-on tests from being built properly.